OpenLiberty/draft-guide-arquillian-managed
# Dev mode

Option 1: simply the pom, but not run in dev mode (current PR content):

- use the ideal(simplified) pom.xml 
- update the Readme content to use dev mode - SEE OPTION 2
  - keep the flow same (let discuss if the flow not work)
- convert `*Test` class to `*IT`
- convert to use JUnit 5 - NOT SUPPORTED 
  - use JUnit5 test order if need
- make sure to correctly configure the ports in test classes, server.xml, and pom.xml
- update travis test
   - `mvn install` cannot be used to verify anymore


Option 2: run guide in dev mode: 

Step to run in dev mode: 
1. Add the `<allowConnectingToRunningServer>true</allowConnectingToRunningServer>` property to the `pom.xml`
2. On a command prompt, run `mvn liberty:dev`
3. On another command prompt, run `mvn liberty:configure-arquillian`
4. On the dev mode prompt, press `enter` to run the tests
